packages/bruno-app/src/utils/importers/openapi-collection.js

```diff
@@ -108,33 +108,34 @@ const transformOpenapiRequestItem = (request) => {
   let auth;
   // allow operation override
   if (_operationObject.security && _operationObject.security.length > 0) {
-    let schemeName = Object.keys(_operationObject.security[0])[0];
-    auth = request.global.security.getScheme(schemeName);
+    auth = request.global.security.getScheme();
   } else if (request.global.security.supported.length > 0) {
-    auth = request.global.security.supported[0];
+    auth = request.global.security.supported;
   }
 
   if (auth) {
-    if (auth.type === 'http' && auth.scheme === 'basic') {
-      brunoRequestItem.request.auth.mode = 'basic';
-      brunoRequestItem.request.auth.basic = {
-        username: '{{username}}',
-        password: '{{password}}'
-      };
-    } else if (auth.type === 'http' && auth.scheme === 'bearer') {
-      brunoRequestItem.request.auth.mode = 'bearer';
-      brunoRequestItem.request.auth.bearer = {
-        token: '{{token}}'
-      };
-    } else if (auth.type === 'apiKey' && auth.in === 'header') {
-      brunoRequestItem.request.headers.push({
-        uid: uuid(),
-        name: auth.name,
-        value: '{{apiKey}}',
-        description: 'Authentication header',
-        enabled: true
-      });
-    }
+    each(auth || [], (param) => {
+      if (param.type === 'http' && param.scheme === 'basic') {
+        brunoRequestItem.request.auth.mode = 'basic';
+        brunoRequestItem.request.auth.basic = {
+          username: '{{username}}',
+          password: '{{password}}'
+        };
+      } else if (param.type === 'http' && param.scheme === 'bearer') {
+        brunoRequestItem.request.auth.mode = 'bearer';
+        brunoRequestItem.request.auth.bearer = {
+          token: '{{token}}'
+        };
+      } else if (param.type === 'apiKey' && param.in === 'header') {
+        brunoRequestItem.request.headers.push({
+          uid: uuid(),
+          name: param.name,
+          value: '{{apiKey}}',
```

Comment:
Hello,
With this line you generate a duplicated variable name, so you need to rework your collection after import.
Maybe a param.name in camelCase can do it?

```js
value = '{{' +
   param.name
             .replace('x-', '')
             .replaceAll('-', ' ')
             .replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {
                  if (+match === 0) return ""; // or if (/\s+/.test(match)) for white spaces
                  return index === 0 ? match.toLowerCase() : match.toUpperCase();
              })
 + '}}';
```

Reply:
Hey , could you please explain how this code generate a duplicated variable name?

Reply:
Ok I apologize it's a mistake, I make confusion on ".name"

```json
"securitySchemes": {
      "apiKey1": {
        "type": "apiKey",
        "name": "x-api-key-1",
        "in": "header"
      },
      "apiKey2": {
        "type": "apiKey",
        "name": "x-api-key-2",
        "in": "header"
      }
    }
```

I thought you use this "name" (with x-api-key-1 / 2) but not.

I checkout your branch and that work fine! I waiting for this merge, I need this fix to :)

Thanks